core/src/main/java/hudson/Functions.java

```diff
@@ -662,18 +662,13 @@ public static String validateIconSize(String iconSize) throws SecurityException
     }
 
     /**
-     * Gets the suffix to use for YUI JavaScript.
+     * @deprecated yui has been removed, do not call this
      */
+    @Deprecated(forRemoval = true, since = "TODO")
```

Comment:
Nice of you to deprecate for a smooth process, but I cannot seem to find any usages outside of core so I think this can be deleted if you wish without any adverse consequences.

Reply:
yes it should work without. yui usage was already removed with https://github.com/jenkinsci/build-monitor-plugin/pull/1023, but seems that one line was missed and https://github.com/jenkinsci/build-monitor-plugin/blob/fec2cf655fe24f4b6372110cb708904c6d083a8c/build-monitor-plugin/src/main/resources/com/smartcodeltd/jenkinsci/plugins/buildmonitor/BuildMonitorView/index.jelly#L43

Reply:
https://github.com/jenkinsci/build-monitor-plugin/pull/1051

Reply:
Released in https://github.com/jenkinsci/build-monitor-plugin/releases/tag/1.14-948.va_a_a_5a_7eb_1f37 so I think we are good to go for removing this.

Reply:
https://github.com/jenkinsci/jenkins/pull/10135/commits/e8c71ee9c591130d67b45a6040825492b111d817